### PR TITLE
Selection page

### PR DESCRIPTION
**Pull request by cmcartaya**

### CODE REVIEW

File: cartaya/selection-page/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" media="screen" href="./resources/css/style.css">
+    <title>XXX3</title>
+</head>
+<body class="view-port">
+    <div class="background">
+        <form class="selection-page_form-container lato-font" action="#" method="GET">
+            <h1 class="selection-page_title">This is the Title</h1>
+            <label class="selection-page_select-container">
+                <input class="selection-page_input" type="radio" name="select" value="excelent">
```
Comment (sarayp) [inline cartaya/selection-page/index.html:15]: there is a typo in excellent across the document.

File: cartaya/selection-page/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                <span class="selection-page_select-order-text">Option 1</span>
+                <span class="selection-page_category-text">Excelent</span>
+            </label>
+            <label class="selection-page_select-container">
+                <input class="selection-page_input" type="radio" name="select" value="excelent">
```
Comment (sarayp) [inline cartaya/selection-page/index.html:20]: the value sent should be good

File: cartaya/selection-page/index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                <span class="selection-page_select-order-text">Option 2</span>
+                <span class="selection-page_category-text">Good</span>
+            </label>
+            <label class="selection-page_select-container">
+                <input class="selection-page_input" type="radio" name="select" value="excelent">
```
Comment (sarayp) [inline cartaya/selection-page/index.html:25]: the value sent should be regular

File: cartaya/selection-page/index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                <span class="selection-page_select-order-text">Option 3</span>
+                <span class="selection-page_category-text">Regulas</span>
```
Comment (sarayp) [inline cartaya/selection-page/index.html:27]: regular has typo

File: cartaya/selection-page/index.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+            </label>
+            <label class="selection-page_select-container">
```
Comment (sarayp) [inline cartaya/selection-page/index.html:29]: the value sent should be bad

File: cartaya/selection-page/resources/css/style.css [hunk 6]
```diff
@@ -0,0 +1,109 @@
+@font-face {
+    font-family: 'Lato-Regular';
+    src: url('./../font/Lato-Regular.eot?#iefix') format('embedded-opentype'),
+         url('./../font/Lato-Regular.woff') format('woff'),
+         url('./../font/Lato-Regular.ttf') format('truetype'),
+         url('./../font/Lato-Regular.svg#FontName') format('svg');
+}
+
```
Comment (sarayp) [inline cartaya/selection-page/resources/css/style.css:8]: extra space

**Review by TahimiLeonBravo — APPROVED** [review]

**Pull request merged**